spec/spec.md

```diff
@@ -149,9 +157,18 @@ The issuance of [[ref: Digital Credential]]s from the [[ref: Issuer]] to the [[r
 OpenID for Verifiable Credential Issuance ([[ref: OID4VCI]]) defines an API for the issuance of [[ref: Digital Credential]]s.
 OID4VCI [issuance flow variations](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID2.html#name-issuance-flow-variations) leave room for optionality.
 
-In many situations, [[ref: Digital Credential]]s are issued on the [[ref: Issuer]]'s online service (website). This online service may have already authenticated and authorized the user before displaying the credential offer. Another authentication or authorization is not needed in those situations. To keep things as simple as possible, DIIP uses *Pre-Authorized Code Flow* instead of *Authorization Code Flow*. (As with other DIIP requirements, [[ref: Issuer]]s are free to issue [[ref: Digital Credential]]s using *Authorization Code Flow*, but then DIIP doesn't guarantee interoperability.)
+In many situations, [[ref: Digital Credential]]s are issued on the [[ref: Issuer]]'s online service (website). This online service may have already authenticated and authorized the user before displaying the credential offer. Another authentication or authorization is not needed in those situations. To keep things as simple as possible, DIIP uses *Pre-Authorized Code Flow*. It should be noted various Security Considerations have been described in the [[ref: OID4VCI]] sspecification with respect to implementing *Pre-Authorized Code Flow*. Parties implementing DIIP are strongly suggested to implement mitigaating measures, like use of a Transaction Code.
 
 **Requirement: DIIP-compliant implementations MUST support the *Pre-Authorized Code Flow*.**
+**Requirement: DIIP-compliant implementations SHOULD support the Transaction Code when using *Pre-Authorized Code Flow*.**
```

Comment:
I think making this a must is fine, and probably good for security. If it doesn't apply to a specific use case it's not required to be used. Just required to be supported